Modules/_winapi.c

```diff
@@ -70,13 +73,22 @@ static BOOL (CALLBACK *Py_CancelIoEx)(HANDLE, LPOVERLAPPED);
 static int
 check_CancelIoEx()
 {
+#ifdef MS_WINDOWS_GAMES
+    Py_CancelIoEx = &CancelIoEx;
+    has_CancelIoEx = TRUE;
+#else
     if (has_CancelIoEx == -1)
     {
-        HINSTANCE hKernel32 = GetModuleHandle("KERNEL32");
-        * (FARPROC *) &Py_CancelIoEx = GetProcAddress(hKernel32,
-                                                      "CancelIoEx");
-        has_CancelIoEx = (Py_CancelIoEx != NULL);
+        HINSTANCE hKernel32 = GetModuleHandleA("KERNEL32");
```

Comment:
What's the issue? The SDK supports `GetModuleHandleW()` and `GetProcAddress()` for `WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP | WINAPI_PARTITION_SYSTEM | WINAPI_PARTITION_GAMES)`. 

That said, `check_CancelIoEx()` and `Py_CancelIoEx` should be removed. `CancelIoEx()` was added to the API in Windows Vista. We haven't supported Windows XP since Python 3.4. `_winapi` can use `CancelIoEx()` directly.

Reply:
I am sure this could work, but would need to get the symbol from a different dll, since the xbox does not use kernel32.dll. So I just went with the simpler solution.

Reply:
Then I will take the chance and just remove it :+1: